geode-dunit/src/main/java/org/apache/geode/cache30/MultiVMRegionTestCase.java

```diff
@@ -3857,10 +3856,25 @@ public void run2() throws CacheException {
       // responses.
     } finally {
       Wait.pause(1500);
+      vm0.invoke("Flush disk store",
+          () -> flushDiskStore(name));
+      vm1.invoke("Flush disk store",
+          () -> flushDiskStore(name));
+      vm2.invoke("Flush disk store",
+          () -> flushDiskStore(name));
       unregisterAllSerializers();
     }
   }
 
+  private void flushDiskStore(String name) {
+    if (getRootRegion() != null && getRootRegion().getSubregion(name) != null) {
+      DiskStoreImpl diskStore = ((LocalRegion) getRootRegion().getSubregion(name)).getDiskStore();
```

Comment:
Please cast to `DiskRecoveryStore` instead of `LocalRegion`. I'm trying to replace all casts to `LocalRegion` to casts with interfaces.